### ISSUE TITLE
DynamicRouteHelper.GetPageUrl invalid column exception

### ISSUE THREAD

**Issue by jkerb135** [KenticoDevTrev/DynamicRouting#9]
When using `DynamicRouteHelper.GetPageUrl` method it is targeting the wrong column in the table. The query is looking for the URL slug by [NodeID] instead of [UrlSlugNodeID]

`SELECT *
FROM DynamicRouting_UrlSlug
WHERE [NodeID] = @NodeID
ORDER BY case when UrlSlugCultureCode = 'en-US' then 0 else 1 end, UrlSlugCultureCode`

File: DynamicRouteInternalHelper.cs 
Method:  `public static string GetPageUrl(int NodeID, string DocumentCulture = null, string SiteName = null)`

Line 1000: 
```
UrlSlugInfo UrlSlug = UrlSlugInfoProvider.GetUrlSlugs()
                .WhereEquals("NodeID", NodeID)
                .OrderBy($"case when UrlSlugCultureCode = '{SqlHelper.EscapeQuotes(DocumentCulture)}' then 0 else 1 end, case when UrlSlugCultureCode = '{CultureHelper.GetDefaultCultureCode(SiteName)}' then 0 else 1 end, UrlSlugCultureCode")
                .FirstOrDefault();
```

**Comment by KenticoDevTrev** [comment]
Got it, give me a couple hours and I'll get this updated and repackaged!

**Comment by KenticoDevTrev** [comment]
Fixed in 26cb186049d6c110dd269bd018733b9c5d957ebb , Nuget Packges for 12.29.3 are uploaded, once NuGet Reindexes then you should be able to see them.

Also please notify me via once you get a chance to try it out, and how hard/easy it was to set up.  http://www.devtrev.com/contact
